keyboards/matrix/pjf/rules.mk

```diff
@@ -0,0 +1,53 @@
+## chip/board settings
+# - the next two should match the directories in
+#   <chibios>/os/hal/ports/$(MCU_FAMILY)/$(MCU_SERIES)
+MCU_FAMILY = STM32
+MCU_SERIES = STM32F1xx
+
+# Linker script to use
+# - it should exist either in <chibios>/os/common/ports/ARMCMx/compilers/GCC/ld/
+#   or <this_dir>/ld/
+MCU_LDSCRIPT = pjf
+
+# Startup code to use
+#  - it should exist in <chibios>/os/common/startup/ARMCMx/compilers/GCC/mk/
+MCU_STARTUP = stm32f1xx
+
+# Board: it should exist either in <chibios>/os/hal/boards/
+#  or <this_dir>/boards
+BOARD = pjf_keyboard
```

Comment:
Is this 100% required, instead of one of the defaults provided in `lib/chibios/os/hal/boards/`?

Reply:
a custom usb connect and disconnect macro was declared on board.h. if there are other place can put them, I can use something like the nucleo boards.

Reply:
> To help with the review, can you confirm the STM32F1 being used please

yes, this board uses stm32f103rb